codegen/templates/endpoint.tmpl

```diff
@@ -250,8 +250,8 @@ func (w {{$workflow}}) Handle(
 }
 
 {{if and (ne .RequestType "") (ne $clientReqType "") -}}
-func convertTo{{title .Name}}ClientRequest(in *{{.RequestType}}) *{{$clientReqType}} {
-	out := &{{$clientReqType}}{}
+func convertTo{{title .Name}}ClientRequest(in {{.RequestType}}) {{$clientReqType}} {
+	out := &{{unref $clientReqType}}{}
```

Comment:
`&string{}` will probably break here ? 

We need some kind of "is primitive" check and not generate converters for non-struct types.